lib/Helper/Mail/Mail.php

```diff
@@ -0,0 +1,270 @@
+<?php
+/**
+  * This helper builds the request body for a /mail/send API call.
+  *
+  * PHP version 5.6, 7
+  *
+  * @author    Elmer Thomas <dx@sendgrid.com>
+  * @copyright 2017 SendGrid
+  * @license   https://opensource.org/licenses/MIT The MIT License
+  * @version   GIT: <git_id>
+  * @link      http://packagist.org/packages/sendgrid/sendgrid
+  */
+namespace SendGrid\Helper\Mail;
+
+/**
+ * The final request body object
+ * Class Mail
+ * @package SendGrid\Helper
+ */
+class Mail implements \JsonSerializable
```

Comment:
Same comment for `Attachment`.

And also, `Public properties`?, why?, this is a very delicate class !, it represents the message that will be send !, let's be more strict and make them private, and type hint people, let's type hint more !!, it doesn't hurt.